test/images/Makefile

```diff
@@ -37,4 +39,7 @@ all-container:
 all-push: all-container
 	./image-util.sh push $(WHAT)
 
-.PHONY: all all-push all-container
+push-manifest: manifest-tool
```

Comment:
is there a reason to make pushing the manifest a separate make target?

Reply:
@ixdy : That was just for separation of concerns within the makefile. However, we can also use all-push target for the same. I have updated my PR accordingly.